app/components/shared/inputs/index.ts

```diff
@@ -85,7 +85,8 @@ const validationMessages = {
       required: () => $t('The field is required'),
       min_value: (fieldName: string, params: number[]) => `The field value must be ${ params[0] } or larger`,
       max_value: (fieldName: string, params: number[]) => `The field value must be ${ params[0] } or less`,
-      date_format: (fieldName: string, params: number[]) => `The date must be in ${ params[0] } format`
+      date_format: (fieldName: string, params: number[]) => `The date must be in ${ params[0] } format`,
+      alpha: () => $t('This field may only contain alphabetic characters')
```

Comment:
Looks like I forgot to use $t here, can you add it for other validation messages pls?